server/test/api/projects.js

```diff
@@ -55,6 +56,34 @@ describe('The /projects API endpoint', function () {
       .catch(helpers.handler);
   });
 
+  it('GET /projects/ ? COMPLETE = 1 returns a complete List of project  ', function () { 
+    return agent.get('/projects?complete=1')
```

Comment:
Can you write a test to verify:
 1. How many projects you expect to receive from the DB with this query?
 2. Which properties you expect to receive from the server?